manifests/extension/runkit.pp

```diff
@@ -0,0 +1,58 @@
+# Installs a php extension for a specific version of php.
+#
+# Usage:
+#
+#     php::extension::runkit { 'runkit for 5.4.10':
+#       php     => '5.4.10',
+#       version => '1.0.3'
+#     }
+#
+define php::extension::runkit(
+  $php,
+  $version = '1.0.3'
+) {
+  include boxen::config
+
+  require php::config
+  # Require php version eg. php::5_4_10
+  # This will compile, install and set up config dirs if not present
+  php_require($php)
+
+  $extension = 'runkit'
+
+  # Final module install path
+  $module_path = "${php::config::root}/versions/${php}/modules/${extension}.so"
+
+  # Clone the source respository
+  repository { "${php::config::extensioncachedir}/runkit":
+    source => 'namesco/runkit'
```

Comment:
The "canonical" source for runkit is... complicated. 

The PECL extension hasn't had a release for 8 years: https://pecl.php.net/package/runkit
Even compiling from source for PECL isn't compatible with PHP 5.3+: http://git.php.net/?p=pecl/php/runkit.git;a=summary
There is a fork on GitHub which works up to PHP 5.5: https://github.com/zenovich/runkit
There is then another fork of that works on PHP 5.6: https://github.com/padraic/runkit

Basically it's a mess, I can request any of these upstreams to create a tag, but none are the canonical source.